components/ChangePercent/ChangePercent.tsx

```diff
@@ -14,21 +15,31 @@ type ChangePercentProps = {
 };
 
 export const ChangePercent: FC<ChangePercentProps> = ({ value, decimals = 2, ...rest }) => {
-	const isPositive = wei(value ?? 0).gt(0);
+	const isValid = value;
+	const isZero = value && wei(value).gt(0);
```

Comment:
I think this should be:
```suggestion
	const isZero = value && wei(value).eq(0);
```